k8_cortx_cloud/destroy-cortx-cloud.sh

```diff
@@ -187,21 +187,6 @@ function deleteCortxLocalBlockStorage()
     uninstallHelmChart "cortx-data-blk-data-${namespace}" "${namespace}"
 }
 
-function deleteCortxPVs()
```

Comment:
The calling of this function was removed in #233. I'm not entirely sure why I removed it, although it doesn't seem it was necessary.